Documentation/markdown/IFC4.3XListOfChanges.md

```diff
@@ -0,0 +1,33 @@
+| IfcCurveSegment derive rules                        | ['Dim']                                                                                                                                                                                                                                                      | []                                                                                                                                                                                                                                        |
+| IfcGradientCurve derive rules                       | ['RelativeElevation']                                                                                                                                                                                                                                        | []                                                                                                                                                                                                                                        |
+| IfcPolygonalBoundedHalfSpace.BoundaryType           | sizeof(typeof(PolygonalBoundary)*['ifc4x3_dev.ifcpolyline','ifc4x3_dev.ifccompositecurve']) = 1                                                                                                                                                              | sizeof(typeof(PolygonalBoundary)*['ifc4x3_dev.ifcpolyline','ifc4x3_dev.ifccompositecurve','ifc4x3_dev.ifcindexedpolycurve']) = 1                                                                                                          |
+| IfcSectionedSurface where rules                     | []                                                                                                                                                                                                                                                           | ['AreaProfileTypes', 'CorrespondingSectionPositions', 'DirectrixIs3D', 'NoOffsets', 'SectionsSameType']                                                                                                                                   |
+| IfcSweptDiskSolidPolygonal.DirectrixIsPolyline      | ('ifc4x3_dev.ifcpolyline' in typeof(self\IfcSweptDiskSolid.Directrix)) or (('ifc4x3_dev.ifcindexedpolycurve' in typeof(self\IfcSweptDiskSolid.Directrix)) and not(exists(self\IfcSweptDiskSolid.Directrix.Segments)))                                        | ('ifc4x3_dev.ifcpolyline' in typeof(self\IfcSweptDiskSolid.Directrix)) or (('ifc4x3_dev.ifcindexedpolycurve' in typeof(self\IfcSweptDiskSolid.Directrix)) and not(exists(self\IfcSweptDiskSolid.Directrix\IfcIndexedPolyCurve.Segments))) |
```

Comment:
Check everything that starts with IfcPoint and also IfcSegment,

Reply:
I added some of them.
IfcPointOnSurface and IfcSegmentDim are not relevant to us or not supported.

Reply:
okay